ui/src/components/MotorInput/BaseMotorInput.jsx

```diff
@@ -71,15 +90,14 @@ function BaseMotorInput(props) {
         max={max}
         min={min}
         disabled={disabled || !isReady}
-        data-dirty={isEdited || undefined}
+        data-invalid={!isInsideLimits || undefined}
+        data-dirty={(isInsideLimits && isEdited) || undefined}
```

Comment:
```suggestion
        data-dirty={isEdited || undefined}
```

Just move the `data-invalid` style after the `data-dirty` style in the CSS so it overrides it.

Reply:
Aha! Changed as suggested.